install_config/aggregate_logging.adoc

```diff
@@ -1462,6 +1462,149 @@ have the same multi-tenant capabilities and your data will not be restricted by
 user access to a particular project.
 ====
 
+[[sending-logs-to-external-rsyslog]]
+== Sending Logs to an External Rsyslog Instance
+
+Use the `fluent-plugin-remote-syslog` plug-in on the host to send logs to an external rsyslog server. 
+
+Set environment variables in the `logging-fluentd` or `logging-mux` deployment configurations:
+
+[source,yaml]
+----
+- name: REMOTE_SYSLOG_HOST <1>
+  value: host1
+- name: REMOTE_SYSLOG_HOST_BACKUP
+  value: host2
+- name: REMOTE_SYSLOG_PORT_BACKUP
+  value: 5555
+----
+<1> Required for each host.
+
+The above will build two destinations. The syslog server on `host1` will be receiving messages on the default port of `514`, while `host2` will be receiving the same messages on port `5555`.
+
+Alternatively, you can configure your own custom `fluent.conf` in the `logging-fluentd` or `logging-mux` ConfigMaps.
+
+[NOTE]
+====
+Any logs on the external rsyslog server will be raw journald output, and not formatted similar to the OpenShift Fluentd output. This means it will not be corrolatable and will have no OpenShift or Kubernetes metadata.
+====
+
+Other usable environment Variables include:
+
+[cols="3,7",options="header"]
+|===
+|Parameter |Description
+
+|`USE_REMOTE_SYSLOG`
+|Defaults to `false`. Set to `true` to enable use of the
+`fluent-plugin-remote-syslog` gem
+
+|`REMOTE_SYSLOG_HOST`
+|(Required) Hostname or IP address of the remote syslog server.
+
+|`REMOTE_SYSLOG_PORT`
+|Port number to connect on. Defaults to `514`.
+
+|`REMOTE_SYSLOG_SEVERITY`
+|Set the syslog severity level. Defaults to `debug`.
+
+|`REMOTE_SYSLOG_FACILITY`
+|Set the syslog facility. Defaults to `local0`.
+
+|`REMOTE_SYSLOG_USE_RECORD`
+|Defaults to `false`. Set to `true` to use the record's severity and facility fields to set on the syslog message.
+
+|`REMOTE_SYSLOG_REMOTE_TAG_PREFIX`
+|Removes the prefix from the tag, defaults to `''` (empty).
+
+|`REMOTE_SYSLOG_TAG_KEY`
+|If specified, uses this field as the key to look on the record, to set the tag on the syslog message.
+
+|`REMOTE_SYSLOG_PAYLOAD_KEY`
+|If specified, uses this field as the key to look on the record, to set the payload on the syslog message.
+
+|===
+
+[WARNING]
+====
+This implementation is insecure, and should only be used in environments
+where you can guarantee no snooping on the connection.
+====
+
+[cols="3,7",options="header"]
+|===
+|Parameter |Description
```

Comment:
I'm not sure what these parameters are for. How are they used?

Reply:
These are the configuration parameters for the gem, exposed via ENV variables.  They're one-to-one with what's described here too.  https://github.com/jkupferer/fluent-plugin-remote-syslog

Take a look at this too https://github.com/sqtran/origin-aggregated-logging/blob/master/docs/remote-syslog.md  Let me know if this helps answer any of your questions.  Definitely ping me again if it doesn't so I can put together more info.